internal/services/firewall/firewall_policy_rule_collection_group_resource.go

```diff
@@ -105,12 +105,15 @@ func resourceFirewallPolicyRuleCollectionGroup() *pluginsdk.Resource {
 										Optional: true,
 										Elem: &pluginsdk.Resource{
 											Schema: map[string]*pluginsdk.Schema{
+												// Mssql is supported but service team didn't include it in the current swagger, they are working on the fix in the next swagger release.
+												// todo: to use the sdk enum for this Mssql const
```

Comment:
can you open a bug on rest specs for them to include it?

Reply:
Sure, will do. I confirmed with them via email and they said they are working on the fix now.